meson.build

```diff
@@ -75,9 +75,8 @@ configinc = include_directories('.')
 cc = meson.get_compiler('c')
 dl_dep = cc.find_library('dl', required : false)
 
-libdrm_dep = dependency('libdrm', version : '>= 2.4.60')
-
-WITH_DRM = not get_option('disable_drm')
+WITH_DRM = not get_option('disable_drm') and (host_machine.system() != 'windows')
+libdrm_dep = dependency('libdrm', version : '>= 2.4.60', required : WITH_DRM)
```

Comment:
This one is off: libdrm_dep is required by all the existing backends - va-drm, va-x11 (+ by extension va-glx) and va-wayland.

Reply:
Thanks for pointing this out. Now I've made libdrm not required only for Windows platforms